src/main/resources/data/households_flanders.json

```diff
@@ -0,0 +1,8768 @@
+[
```

Comment:
So *this* is why GitHub says `+9,256/−434` about this PR. I must say I'm relieved. I was kinda worried that I'd be reviewing code until midnight.